compiler/catala_utils/file.ml

```diff
@@ -180,6 +180,42 @@ let process_out ?check_exit cmd args =
     assert false
   with End_of_file -> Buffer.contents buf
 
+(* SIDE EFFECT AT MODULE LOAD: sets up a signal handler on SIGWINCH (window
```

Comment:
This is very fancy

Reply:
Well I had the code lying around, so... 🤷🏿